numismatic/feeds/poloniex.py

```diff
@@ -47,9 +40,49 @@ def handle_trades_and_orders(msg, subscription):
             one message with both information embedded
         '''
 
-        PoloniexWebsocketClient._handle_orderbook(msg, subscription)
-        PoloniexWebsocketClient._handle_trade(msg, subscription)
-        return STOP_HANDLERS
+        # FIXME: Should probably loop through messages
+        # instead of calling them by index
+        if len(msg) == 3:
+            if len(msg[2]) == 1:
+                PoloniexWebsocketClient._handle_order_book(msg, subscription)
+                PoloniexWebsocketClient._handle_orderbook_removemodify(msg, subscription)
+            elif len(msg[2]) == 2:
+                PoloniexWebsocketClient._handle_orderbook_removemodify(msg, subscription)
+                PoloniexWebsocketClient._handle_trade(msg, subscription)
+
+            return STOP_HANDLERS
+
+    @staticmethod
+    def _handle_order_book(msg, subscription):
+        '''
+            Poloniex order book, this will provide a snapshot
+            of the order book
+            TODO Put format here
+        '''
+
+        if msg[2][0][0] == 'i':
+            raw_data = msg[2][0][1]
+
+            for key, value in raw_data.items():
+                if key == 'orderBook':
+                    for ask_price, volume in value[0].items():
+                        event = Order(
```

Comment:
You could keep the `order_event` name. That was descriptive.

Reply:
I kinda disagree. You should get the context from the method name and perhaps the type it represents. `event` seems descriptive, but if we just stick to event everywhere throughout the codebase, it will be easier, as opposed to always thinking of new variable names for these events.